csharp-basic-topics/ActionAndFuncDelegates/ActionAndFuncDelegates/Program.cs

```diff
@@ -0,0 +1,122 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace CodeMaze
+{
+    public class Program
+    {
+        public static void Main(string[] args)

```

Comment:
You can use file scoped namespaces, global using and remove the main method since it's .NET 6. It's up to you of course, but just to remind you that we support that.

Reply:
Well, I was aware of this flexibility introduced in .NET6, but then thought that I might need to keep this project backward compatible. Now, I agree with you. Since we are setting target framework as .NET6, there is no point in backward compatibility. I will change this shortly.

Reply:
You have a point on backward compatibility, but it's easy to change it nowadays with dotnet format. It's really up to you, I just wanted to tell you that you can do it like this if you want.